README.md

```diff
@@ -79,7 +116,14 @@ sudo usermod -a -G microk8s $USER
 sudo chown -f -R $USER ~/.kube
 microk8s enable ingress dns storage rbac gpu
 microk8s enable host-access:ip=10.0.1.20
-# add a line `10.0.1.20<TAB>tensorleap.local` to `/etc/hosts`
+echo "10.0.1.20\ttensorleap.local" | sudo tee -a /etc/hosts
+
+# Get a service account key file with read permissions to storage and add to cluster
+kubectl --kubeconfig=<(microk8s config) create secret docker-registry gcr-access-token \
+  --docker-server=gcr.io \
+  --docker-username=_json_key \
+  --docker-password="$(cat ./json-key-file.json)" \
```

Comment:
I don't think that the `./json-key-file.json` is created in this section...

Reply:
That's why the comment

Reply:
Ohhhh